src/libraries/System.Linq/src/System/Linq/Select.SpeedOpt.cs

```diff
@@ -697,18 +702,33 @@ public override TResult[] ToArray()
                 };
             }
 
+            private List<TResult> LazyToList()
```

Comment:
Nit: with "lazy" I presume you mean to list without presizing? I don't think that's an accurate use of the term.
```suggestion
            private List<TResult> ToListNoPresizing()
```

Reply:
TBH I just copied `private TResult[] LazyToArray()` on line 665. Should I change both?

Reply:
Ah fair. Thanks for improving it.